src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpLogExporter.cs

```diff
@@ -19,7 +19,7 @@ namespace OpenTelemetry.Exporter;
 /// </summary>
 public sealed class OtlpLogExporter : BaseExporter<LogRecord>
 {
-    private readonly IExportClient<OtlpCollector.ExportLogsServiceRequest> exportClient;
+    private readonly OtlpExporterTransmissionHandler<OtlpCollector.ExportLogsServiceRequest> transmissionHandler;
```

Comment:
nit: consider prefixing signal type on this variable, for e.g. `traceTransmissionHandler`.